### PR TITLE
Feat/support for containers

### PR DESCRIPTION
**Pull request by vladiulianbogdan**
# Pull Request Template

## Type of change

-   [x] 🍕 New feature
-   [ ] 🐛 Bug Fix
-   [ ] 🔥 Breaking change
-   [ ] 🧑‍💻 Improvement
-   [ ] 📝 Documentation Update

## Description

Add support for deploying containers.

## Checklist

-   [x] My code follows the contributor guidelines of this project;
-   [ ] I have updated the documentation;
-   [ ] I have added tests;
-   [x] New and existing unit tests pass locally with my changes;

### CODE REVIEW

File: src/constants.ts
Comment (costinsin) [inline src/constants.ts:1]: Drop this change

File: package.json [hunk 1]
```diff
@@ -126,6 +126,7 @@
         "@types/cli-table": "^0.3.4",
         "@types/cors": "^2.8.17",
         "@types/decompress": "^4.2.7",
+        "@types/dockerode": "^3.3.31",
```
Comment (vladiulianbogdan) [inline package.json:129]: I need to remove this

File: src/commands/deploy/command.ts [hunk 2]
```diff
@@ -83,5 +89,9 @@ function decideDeployType(): DeployType {
         }
     }
 
+    if (fs.existsSync("Dockerfile")) {
```
Comment (stefandarius) [inline src/commands/deploy/command.ts:92]: this means the CLI will assume a docker deployment whenever it detects a Dockerfile? if I would like to switch to classic deployment I should just delete de Dockerfile?

Comment (vladiulianbogdan) [reply src/commands/deploy/command.ts:92]: Yea, or move the Dockerfile or rename it... it is not ideal, but we wanted to get this started.

**Pull request merged**